src/util.h

```diff
@@ -809,10 +809,40 @@ class CInstPictures
 class CLocale
 {
 public:
-    static QString                 GetCountryFlagIconsResourceReference ( const QLocale::Country eCountry );
+    static QString                 GetCountryFlagIconsResourceReference ( const QLocale::Country eCountry /* Always a Qt5 (!) code */ );
     static QMap<QString, QString>  GetAvailableTranslations();
     static QPair<QString, QString> FindSysLangTransFileName ( const QMap<QString, QString>& TranslMap );
     static void                    LoadTranslation ( const QString strLanguage, QCoreApplication* pApp );
+    static QLocale::Country        WireFormatCountryCodeToQtCountry ( unsigned short usCountryCode );
+    static unsigned short          QtCountryToWireFormatCountryCode ( const QLocale::Country eCountry );
+    static bool                    IsCountryCodeSupported ( unsigned short iCountryCode );
+#if QT_VERSION >= QT_VERSION_CHECK( 6, 0, 0 )
+    // ./tools/qt5-to-qt6-country-code-table.py generates these lists:
+    constexpr int const static qt5CountryToQt6Table[] = {
```

Comment:
Can we rename these `wireFormatTo...` and `...ToWireFormat`?

Reply:
Done.